node/events_test.ts

```diff
@@ -0,0 +1,1138 @@
+import { assert } from "../testing/asserts.ts";
```

Comment:
nit: please add the license header
```suggestion
// Copyright 2018-2022 the Deno authors. All rights reserved. MIT license.
import { assert } from "../testing/asserts.ts";
```